src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const {
+    slug,
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  } = person;
+  const { personSlug = '' } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': personSlug === slug,
+      })}
+    >
+      <td>
+        <PersonLink person={person} />
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {(
+          mother
+            ? <PersonLink person={mother} />
+            : motherName || '-'
```

Comment:
ternary operator should return one value depending on condition, don't make another condition in the ternary operator